sci-geosciences/dcw-gmt-data/dcw-gmt-data-1.1.4.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="The Digital Chart of the World for GMT 5 or later"
+HOMEPAGE="https://www.soest.hawaii.edu/wessel/dcw/"
+SRC_URI="https://www.soest.hawaii.edu/pwessel/dcw/dcw-gmt-${PV}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
+S="${WORKDIR}"
+
+src_install() {
+	dodoc dcw-gmt-${PV}/{{LICENSE,README}.TXT,COPYING.LESSERv3,COPYINGv3,ChangeLog}
```

Comment:
Ok, I use MY_P and change S.

The suffix is added to emphasize that this is data, not programs. I can rename dcw-gmt-data to dcw-gmt and gshhg-data to gshhg-gmt, if you think that it will be better.

As I understand it, the LICENSE.txt is necessary? Only COPYING.LESSERv3 and COPYINGv3 are not needed?

Reply:
You don't need to emphasize it. People can read descriptions.

I don't think LICENSE.txt is needed. The ebuild states the license, and Portage handles users accepting or not accepting particular licenses.

Reply:
Ok, I renamed ebuilds and removed LICENSE.txt.